evmd/go.mod

```diff
@@ -280,7 +280,7 @@ replace (
 	github.com/cosmos/evm => ../
 	github.com/cosmos/evm/evmd => ./evmd
 	// use Cosmos geth fork
-	github.com/ethereum/go-ethereum => github.com/cosmos/go-ethereum v1.15.11-cosmos-0
+	github.com/ethereum/go-ethereum => github.com/yihuang/go-ethereum v1.10.21-0.20250721081216-98795aa589d5
```

Comment:
https://github.com/yihuang/go-ethereum/tree/cosmos-1-16

Reply:
Could you upstream your changes to `cosmos/go-ethereum`?  We would be happy to integrate

Reply:
we need to probably create a base `release/1.16` branch first?

Reply:
@yihuang I created https://github.com/cosmos/go-ethereum/tree/release/1.16 off of [v1.16.2](https://gitkraken.dev/link/Z2l0a3Jha2VuOi8vcmVwb2xpbmsvNWRiMzMzNWRjZTc2NmJkNjc5YzU0ZWE0NGY2ZGYwOGE3ZmY3NDc2Mi90YWcvdjEuMTYuMj91cmw9Z2l0JTQwZ2l0aHViLmNvbSUzQWV0aGVyZXVtJTJGZ28tZXRoZXJldW0uZ2l0?origin=gitkraken). Feel free to make a PR off of that :)